packages/api/src/tx/modules/ecocredit/v1/send_amino.ts

```diff
@@ -0,0 +1,74 @@
+import { AminoMsg } from '@cosmjs/amino';
+import { AminoConverter } from '@cosmjs/stargate';
+import {
+  MsgSend,
+  MsgSend_SendCredits,
+} from '../../../../generated/regen/ecocredit/v1/tx';
+
+const msgSendAminoType = 'regen.core/MsgSend';
+
+export const sendTypeUrl = '/' + MsgSend.$type;
+
+interface AminoMsgSend_SendCredits {
+  batch_denom: string;
+  tradable_amount?: string;
+  retired_amount?: string;
+  retirement_jurisdiction?: string;
+}
+
+export interface AminoMsgSend extends AminoMsg {
+  readonly type: typeof msgSendAminoType;
+  readonly value: {
+    // readonly $type: string; TODO: we will leave these off until nested types can be supported
```

Comment:
Only leaving a note here to avoid a ton of comments. For this solution, we must omit any nested Meg Types in amino messages. Without this change, we get the unauthorized error.

When the ledger is upgraded, and maybe when cosmjs settles on a way to handle this, we can put the nested msg types back in.

cc @amaurym